knowledge-base/grid-scroll-to-selected-row.md

```diff
@@ -22,11 +22,266 @@ res_type: kb
 
 
 ## Description
-I would like to pre-select a row when the page is loaded, and I would like to show the grid with that row scrolled to the top, or sorted to appear on top.
+
+I want to [select a row in Grid]({%slug grid-selection-row%}) programmatically based on some program logic and conditions. I want after that to scroll the Grid to the selected row so the user can see the selected row.
```

Comment:
```suggestion
I want to programmatically [select a row in the Grid]({%slug grid-selection-row%}) based on specific conditions in my code. Once selected, I’d like the Grid to automatically scroll to that row so it’s visible to the user.
```
I suggest this change, because in my opinion the description would be more fluent and natural.